doc/Makefile

```diff
@@ -160,3 +160,6 @@ linkcheck: version-check
 show:
 	@python -c "import webbrowser; webbrowser.open_new_tab('file://$(PWD)/build/html-scipyorg/index.html')"
 
+
+livehtml:
+	sphinx-autobuild -b html $(ALLSPHINXOPTS) build/html $(0)
```

Comment:
Might remove - sphinx-autobuild is greedy and slow for these docs! Favouring liveserve for now

Reply:
Removing now - liveserve works well and the docs are built in the workspace prebuild